explanation/intro-to/ebpf.md

```diff
@@ -13,26 +13,25 @@ has access to the kernel data structures, providing more capabilities than
 tools limited to the interfaces exposed to user-space.
 
 BPF, which stands for "Berkeley Packet Filter", was originally designed to
-perform network packet filtering. Over time, its capabilities were extended far
-beyond that scope -- this is reflected in the current name, *extended Berkeley
-Packet Filter* (eBPF). It now uses more registers, supports 64-bit registers,
-data stores (Maps), and more.  Such enhancements allowed eBPF to be decoupled
-from the kernel networking subsystem and transformed it into a tool with
-broader scope, capable of enhancing not only the networking experience, but
-also tracing, profiling, observability, security, etc.  The terms eBPF and BPF
-are now used interchangeably, but refer to eBPF -- which has become a
-standalone term that no longer refers to *extended Berkeley Packet Filter* due
-to its aforementioned broader scope.
-
-## How it works
+perform network packet filtering. Over time, it has evolved into *extended Berkeley
+Packet Filter* (eBPF), a tool which contains many additional capabilities, including 
+the use of more registers, support for 64-bit registers, data stores (Maps), and 
+more.
+
+As a result, eBPF has been extended beyond the kernel networking subsystem and 
+it not only enhances the networking experience, but also provides tracing, 
+profiling, observability, security, etc.  The terms eBPF and BPF are  
+used interchangeably but both refer to eBPF now.
```

Comment:
```suggestion
profiling, observability, security, etc. The terms eBPF and BPF are used interchangeably but both refer to eBPF now.
```

Reply:
The rendered page had a weird break in it - not sure why, but hopefully this will fix it